app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command_list = command.split(" ")
+    if "/" in command_list[2]:
+        index_new_directories = command_list[2].rfind("/")
+        new_directories = command_list[2][:index_new_directories]
+        os.makedirs(new_directories, exist_ok=True)
+
+    if command_list[0] == "mv":
+        with (
+            open(command_list[1], "r") as file_riad,
```

Comment:
The variable name `file_riad` seems to be a typo. It should likely be `file_read` to indicate that this file object is used for reading.